database/reset_schema.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS trains;
```

Comment:
Thanks for bringing this up @pllearns . We will get rid of the schema.sql file. The reset_schema.sql helps delete the tables and recreates them.